src/main/scala/cromwell/engine/EngineFunctions.scala

```diff
@@ -1,18 +1,22 @@
 package cromwell.engine
 
 import cromwell.binding.WdlFunctions
-import cromwell.binding.values.{WdlInteger, WdlString, WdlValue}
+import cromwell.binding.values.{WdlFile, WdlInteger, WdlString, WdlValue}
 
 import scala.util.Try
 
 trait EngineFunctions extends WdlFunctions {
   protected def read_int(params: Seq[Try[WdlValue]]): Try[WdlInteger]
   protected def read_string(params: Seq[Try[WdlValue]]): Try[WdlString]
+  protected def stdout(params: Seq[Try[WdlValue]]): Try[WdlFile]
+  protected def stderr(params: Seq[Try[WdlValue]]): Try[WdlFile]
 
   def getFunction(name: String): WdlFunction = {
     name match {
       case "read_int" => read_int
       case "read_string" => read_string
+      case "stdout" => stdout
```

Comment:
Actually on further reflection I think I'm going to suggest we keep things as they are.  I like that required engine functions are spelled out in the trait so new backends will know exactly what needs to be implemented with compile-time checking.  And while lower camel case is certainly the rule in Scala, in this case I feel like I'd be mentally mapping from Pythonic to lower camel for no reason other than absolute conformity to that rule.